Sources/BoltsSwift/Task+ContinueWith.swift

```diff
@@ -0,0 +1,195 @@
+/*
+ *  Copyright (c) 2016, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import Foundation
+
+extension Task {
+    // MARK: Continuations
```

Comment:
Nit: Remove this mark, as it's pretty much useless 😁 

Also, might be a good idea to split `continueWith`/`continueOnSuccess`/`continueOnError` into separate extensions and add marks for all of them that designate sections. Should improve navigation in this source file, or at least I hope so...